zerver/views/registration.py

```diff
@@ -539,38 +539,33 @@ def find_account(request: HttpRequest) -> HttpResponse:
     url = reverse('zerver.views.registration.find_account')
 
     emails = []  # type: List[str]
+    emails_sent = []  # type: List[str]
+    emails_ignore = []  # tyep: List[str]
     if request.method == 'POST':
         form = FindMyTeamForm(request.POST)
         if form.is_valid():
             emails = form.cleaned_data['emails']
-            for user in UserProfile.objects.filter(
-                    delivery_email__in=emails, is_active=True, is_bot=False,
-                    realm__deactivated=False):
-                context = common_context(user)
-                context.update({
-                    'email': user.delivery_email,
-                })
-                send_email('zerver/emails/find_team', to_user_ids=[user.id], context=context,
+            for user in UserProfile.objects.filter(delivery_email__in=emails):
+                if user.is_active is True and user.is_bot is False and user.realm.deactivated is False:
```

Comment:
Just write `if user.is_active`; `is True` is a weird way to write things.

Reply:
Okay! will do it in the next push :+1: